### ISSUE TITLE
Review & Groom Backlog

### ISSUE THREAD

**Issue by hleckenb** [bcgov/ckanext-bcgov#587]
As a project team we need to understand backlog priorities and ensure tickets have enough information to proceed with work.

DoD
Review all backlog tickets for clarity.
Assign a prioritization indicator to help determine which work should be completed first.
Review assignment of release numbers. If not part of the next release (1.7.10) remove release number and reassign to backlog.
